src/Artsy/Router/buildServerApp.tsx

```diff
@@ -107,14 +130,38 @@ export function buildServerApp(config: ServerRouterConfig): Promise<Resolve> {
             )
           }
 
+          // Build up script tags to inject into head
+          const scripts = []
+
           const { relayData: _relayData, styleTags, bodyHTML } = await trace(
             "buildServerApp.fetch",
             (async () => {
               const sheet = new ServerStyleSheet()
 
-              const html = ReactDOMServer.renderToString(
-                sheet.collectStyles(<ServerApp tags={headTags} />)
-              )
+              // Extract bundle split chunks for SSR
+              let bundleJSX
+
+              // FIXME: Remove once EXPERIMENTAL_APP_SHELL A/B test completes
+              if (getENV("EXPERIMENTAL_APP_SHELL")) {
+                const extractor = new ChunkExtractor({
+                  statsFile,
+                  entrypoints: ["experimental-app-shell"],
```

Comment:
Currently, the only webpack chunk that supports SSR bundle-splitting is code within the new app shell hot path. Since all new apps will eventually be mounted here, we can start aggressively bundle splitting other code, and further reduce main bundle sizes. For example -- subroutes within the artist page (the tabs) can fetched only when the user clicks the tab, reducing the main bundle size.